<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CANview</class>
 <widget class="QDialog" name="CANview">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="Connect">
      <attribute name="title">
       <string>Connect</string>
      </attribute>
      <widget class="QCheckBox" name="connectPort">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputNamePort">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>COM66</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>621</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextBrowser" name="TX_textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>271</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="RX_textBrowser_2">
       <property name="geometry">
        <rect>
         <x>345</x>
         <y>80</y>
         <width>281</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputNamePort</tabstop>
  <tabstop>connectPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
